test/core/fixtures/transformation/es6.template-literals/expression-first/expected.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+var t = "" + foo + "bar";
+
+var foo = 5;
+var bar = 10;
+var baz = 15;
+
+var example = 1 + ("" + foo + bar + baz);
```

Comment:
> If `bar` and `baz` are both numbers then they'll be added together instead of being coerced to strings.

You mean in `"" + foo + bar + baz`? No, `+` left-to-right associative. You only need a string at the beginning.